<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="inputLabel">
       <property name="text">
        <string>Input Image :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="inputPushButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Browser</string>
       </property>
       <property name="autoRepeatInterval">
        <number>98</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>701</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="outputLabel">
       <property name="text">
        <string>Output Image :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="outputPushButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Browser</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="displayImageCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>679</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>92</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Display Image After Saving</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="filterTypeGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>700</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter type</string>
    </property>
    <widget class="QRadioButton" name="gaussianBlurRatioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Gaussian Blur</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="medianBlurRadioButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Median Blur</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionEnglish"/>
    <addaction name="actionGerman"/>
    <addaction name="actionTurkish"/>
   </widget>
   <addaction name="menuLanguage"/>
  </widget>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionGerman">
   <property name="text">
    <string>German</string>
   </property>
  </action>
  <action name="actionTurkish">
   <property name="text">
    <string>Turkish</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
